### PR TITLE
ja/docs/reference/command-line-tools-reference/feature-gates.md

### LINKED ISSUE #28272: ja: Make docs/reference/command-line-tools-reference/feature-gates.md follow v1.19 of the original text
**This is a Feature Request**

**What would you like to be added**
Update docs/reference/command-line-tools-reference/feature-gates.md to follow v1.19 of the original (English) text.

**Why is this needed**
content/ja/docs/reference/command-line-tools-reference/feature-gates.md is outdated.

**Comments**
/language ja
/good-first-issue

File to update
https://github.com/kubernetes/website/tree/dev-1.19-ja.2/content/ja/docs/reference/command-line-tools-reference/feature-gates.md

Original
https://github.com/kubernetes/website/tree/release-1.19/content/en/docs/reference/command-line-tools-reference/feature-gates.md

diff between translated and v1.19
https://gist.github.com/5dc5c02e8a00e393d60fc5e4c66f6cc7

**Note**
Currently, we use **dev-1.19-ja.2** branch. Please open a PR targeting the branch.

ref
https://kubernetes.io/ja/docs/contribute/localization/

If you have a question, feel free to ask us at slack.k8s.io #kubernetes-docs-ja channel

### PR DESCRIPTION
**Pull request by s-ito-ts**
Update ja/docs/reference/command-line-tools-reference/feature-gates.md

Fixes #28272

### CODE REVIEW

File: content/ja/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -327,13 +346,13 @@ GAになってからさらなる変更を加えることは現実的ではない
 - `AnyVolumeDataSource`: {{< glossary_tooltip text="PVC" term_id="persistent-volume-claim" >}}の`DataSource`としてカスタムリソースの使用を有効にします。
 - `AllowExtTrafficLocalEndpoints`: サービスが外部へのリクエストをノードのローカルエンドポイントにルーティングできるようにします。
 - `APIListChunking`: APIクライアントがAPIサーバーからチャンク単位で（`LIST`や`GET`の）リソースを取得できるようにします。
-`APIPriorityAndFairness`: 各サーバーで優先順位付けと公平性を備えた要求の並行性を管理できるようにします(`RequestManagement`から名前が変更されました)。
+- `APIPriorityAndFairness`: 各サーバーで優先順位付けと公平性を備えた要求の並行性を管理できるようにします(`RequestManagement`から名前が変更されました)。
 - `APIResponseCompression`:`LIST`や`GET`リクエストのAPIレスポンスを圧縮します。
-- `AppArmor`: Dockerを使用する場合にLinuxノードでAppArmorによる強制アクセスコントロールを有効にします。詳細は[AppArmorチュートリアル](/docs/tutorials/clusters/apparmor/)で確認できます。
-- `AttachVolumeLimit`: ボリュームプラグインを有効にすることでノードにアタッチできるボリューム数の制限を設定できます。
+- `AppArmor`: Dockerを使用する場合にLinuxノードでAppArmorによる強制アクセスコントロールを有効にします。詳細は[AppArmorチュートリアル](/ja/docs/tutorials/clusters/apparmor/)で確認できます。
```
Comment (s-ito-ts) [inline content/ja/docs/reference/command-line-tools-reference/feature-gates.md:351]: The link target has been translated into Japanese, so I changed the link.

File: content/ja/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -351,33 +370,41 @@ GAになってからさらなる変更を加えることは現実的ではない
...
 - `CustomResourceDefaulting`: OpenAPI v3バリデーションスキーマにおいて、デフォルト値のCRDサポートを有効にします。
 - `CustomResourcePublishOpenAPI`: CRDのOpenAPI仕様での公開を有効にします。
-- `CustomResourceSubresources`: [CustomResourceDefinition](/docs/concepts/extend-kubernetes/api-extension/custom-resources/)から作成されたリソースの`/status`および`/scale`サブリソースを有効にします。
-- `CustomResourceValidation`: [CustomResourceDefinition](/docs/concepts/extend-kubernetes/api-extension/custom-resources/)から作成されたリソースのスキーマによる検証を有効にします。
-- `CustomResourceWebhookConversion`: [CustomResourceDefinition](/docs/concepts/extend-kubernetes/api-extension/custom-resources/)から作成されたリソースのWebhookベースの変換を有効にします。
+- `CustomResourceSubresources`: [CustomResourceDefinition](/ja/docs/concepts/extend-kubernetes/api-extension/custom-resources/)から作成されたリソースの`/status`および`/scale`サブリソースを有効にします。
+- `CustomResourceValidation`: [CustomResourceDefinition](/ja/docs/concepts/extend-kubernetes/api-extension/custom-resources/)から作成されたリソースのスキーマによる検証を有効にします。
+- `CustomResourceWebhookConversion`: [CustomResourceDefinition](/ja/docs/concepts/extend-kubernetes/api-extension/custom-resources/)から作成されたリソースのWebhookベースの変換を有効にします。
```
Comment (s-ito-ts) [inline content/ja/docs/reference/command-line-tools-reference/feature-gates.md:386]: The link target has been translated into Japanese, so I changed the link.

File: content/ja/docs/reference/command-line-tools-reference/feature-gates.md [hunk 3]
```diff
@@ -268,9 +290,6 @@ content_type: concept
 | `VolumeScheduling` | `false` | Alpha | 1.9 | 1.9 |
 | `VolumeScheduling` | `true` | Beta | 1.10 | 1.12 |
 | `VolumeScheduling` | `true` | GA | 1.13 | - |
-| `VolumeScheduling` | `false` | Alpha | 1.9 | 1.9 |
```
Comment (atoato88) [inline content/ja/docs/reference/command-line-tools-reference/feature-gates.md:271]: Nice catch of removing this section! :smile:

**Comment by s-ito-ts** [comment]
/assign @inductor
Could you approve this PR?

**Pull request merged**